test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
The behavior of `inject` is best illustrated in the new test file. Here is an excerpt.

```elixir
    defmodule NoTopLevel do
      use Protobuf, from: [Path.expand("../proto/basic.proto", __DIR__)],
                    inject: true
    end
    assert %{f2: "I do what I want"} = Basic.new(f2: "I do what I want")
```

The `Basic` message is given a module name of `Basic` instead of `NoTopLevel.Basic`. So the package namespacing is used, but the module that is calling `use Protobuf` is not used at all in the naming.

I think the current usage of `inject` is about modifying an existing module rather than redefining that module and the new behavior really doesn't have anything to do with that which is why I'm wondering about maybe separating this new behavior to its own flag.